### PR TITLE
Fix rule admission_control_plugin_SecurityContextDeny for ocp4

### PR DESCRIPTION
**Pull request by ralvares**
Update rule admission_control_plugin_SecurityContextDeny - CIS 1.2.13 to support ocp4

### CODE REVIEW

File: applications/openshift/api-server/api_server_admission_control_plugin_SecurityContextDeny/rule.yml [hunk 1]
```diff
@@ -19,22 +23,56 @@ description: |-
...
+
+template:
+  name: yamlfile_value
+  vars:
+    ocp_data: "true"
+    entity_check: "none satisfy"
+    filepath: /api/v1/namespaces/openshift-kube-apiserver/configmaps/config
+    yamlpath: '.data["config.yaml"]'
+    values:
+    - value: '"enable-admission-plugins":\[[^]]*"SecurityContextConstraint"'
```
Comment (JAORMX) [inline applications/openshift/api-server/api_server_admission_control_plugin_SecurityContextDeny/rule.yml:76]: This one is a little confusing to me. Isn't this about SecurityContextDeny?

Comment (ralvares) [reply applications/openshift/api-server/api_server_admission_control_plugin_SecurityContextDeny/rule.yml:76]: If my understanding of the documentation is correct, For 4.6 and above, verify that the set of admission plugins includes SecurityContextConstraint, is that correct ?

Comment (redhatrises) [reply applications/openshift/api-server/api_server_admission_control_plugin_SecurityContextDeny/rule.yml:76]: This rule is about SecurityContextDeny and not SecurityContextConstraint. SecurityContextConstraint rules are found under `applications/openshift/scc`. If we need to verify that SecurityContextConstraint is enabled, we need a new rule.

Comment (ralvares) [reply applications/openshift/api-server/api_server_admission_control_plugin_SecurityContextDeny/rule.yml]: Fixed!

**Review by JAORMX — APPROVED** [review]

**Comment by jhrozek** [comment]
We can either merge the PR now as-is or drop the ocp4 conditionals since they are no longer needed since we dropped ocp3 recently.

**Comment by JAORMX** [comment]
I'll merge this and fix this up in a separate PR.

**Pull request merged**